tests/scenarios/personas.spec.js

```diff
@@ -0,0 +1,112 @@
+      const videoBtnSelector = `${selector} [data-apos-add-item=apostrophe-video]`;
+      const videoInput = `.apos-field-input[name=video]`;
+      const videoUrl = 'https://www.youtube.com/watch?v=OV1yagf8Uqk';
+      const save = '[data-apos-save]';
+      client.getLocationInView(selector);
+      client.moveToElement(addContentBtnSelector, 0, 0);
+      client.waitForElementReady(addContentBtnSelector);
+      client.clickWhenReady(addContentBtnSelector);
+      client.waitForElementReady(videoBtnSelector);
+      client.pause(200);
+      client.clickWhenReady(videoBtnSelector);
+      client.waitForModal('apostrophe-video-widgets-editor');
+      client.resetValueInModal('apostrophe-video-widgets-editor', videoInput, videoUrl);
+      client.pause(3000);
+      client.clickInModal('apostrophe-video-widgets-editor', save);
+      client.waitForNoModals();
+    }
+  },
+  {
+    'Switch to human persona': function(client) {
+      const aposUi = '[data-apos-widget-wrapper=apostrophe-video] .apos-area-widget-controls'
+      const personaSelect = `${aposUi} .apos-button[name=personas]`;
+      const humanPersonaValue = `${personaSelect} option[value="human"]`;
+      client.execute(function(aposUi) {
+        $(aposUi).css({opacity: '1'});
+      },[aposUi]);
+      client.moveToElement(personaSelect, 0, 0);
+      client.clickWhenReady(personaSelect);
+      client.clickWhenReady(humanPersonaValue);
+    }
+  },
+  {
+    'Hide video widget to avoid accidently playing it when submitting': function(client) {
+      const aposUi = '[data-apos-widget-wrapper=apostrophe-video]'
+      client.execute(function(aposUi) {
+        $(aposUi).css({display: 'none'});
+      },[aposUi]);
+    }
+  },
+  steps.submitChanges(),
+  steps.commitAndExport(),
+  {
+    'log out of Apostrophe': function(client) {
+      client.openAdminBarItem('apostrophe-login-logout')
+    }
+  },
+  steps.navigateToRelativeUrl('robot/personas-test'),
+  {
+    'should have the Rich Text widget in robot persona': function(client) {
+      const richTextSelector = `.demo-main [data-rich-text]`;
+      client.expect.element(richTextSelector).text.to.contain('Rich Text Widget Robot Line');
+    }
+  },
+  steps.navigateToHome(),
+  steps.navigateToRelativeUrl('human/personas-test'),
+  {
+    'should have the Video widget in human persona': function(client) {
+      const videoWidget = `.demo-main [data-apos-widget-wrapper=apostrophe-video]`;
+      client.expect.element(videoWidget).to.be.present;
+    }
+  },
+  steps.navigateToHome(),
+  steps.navigateToRelativeUrl('personas-test'),
+  {
+    'should not have any widgets in universal persona': function(client) {
+      const richTextSelector = `.demo-main [data-rich-text]`;
+      const videoWidgetSelector = `.demo-main [data-apos-widget-wrapper=apostrophe-video]`;
+      client.expect.element(richTextSelector).to.not.be.present;
+      client.expect.element(richTextSelector).to.not.be.present;
```

Comment:
You are testing the rich text selector twice.

Also as near as I can tell *both* widgets should be present in the universal persona. Are you sure they aren't (are you sure the test is correct)? If they aren't then a significant personas bug might need fixing.

Reply:
Per the docs all widgets should be visible in the universal persona, because the user's preference is not known yet.

Reply:
Interesting. I can confirm the following behavior manually:

**When logged-out**
- Visiting `/en/robot/personas-test` will only display the rich text widget.
- Visiting `/en/human/personas-test` will only display the video widget.
- Visiting `/en/personas-test` displays no widgets. 

**`live` mode**
- Visiting `/en/robot/personas-test` will display both widgets.
- Visiting `/en/human/personas-test` will display both widgets.
- Visiting `/en/personas-test` will display both widgets.

**`draft` mode**
- Visiting `/en/robot/personas-test` will display the rich-text widget, and an opaque video widget. 
- Visiting `/en/human/personas-test` will display the video widget, and an opaque rich-text widget.
- Visiting `/en/personas-test` will display both widgets, but opaque.